priv/repo/migrations/20161020144622_create_stripe_account.exs

```diff
@@ -0,0 +1,28 @@
+defmodule CodeCorps.Repo.Migrations.CreateStripeAccount do
+  use Ecto.Migration
+
+  def change do
+    create table(:stripe_accounts, primary_key: false) do
```

Comment:
It appears we're telling `create` to not create the traditional primary key `id` but then we add and set the `id` column below to `primary_key: true`. Perhaps we can simplify this to `create table(:stripe_accounts) do` and remove `add :id, :string, primary_key: :true` as the `id` column will be created for us.